source/extensions/filters/http/adaptive_concurrency/controller/gradient_controller.cc

```diff
@@ -132,7 +132,8 @@ std::chrono::milliseconds GradientController::applyJitter(std::chrono::milliseco
     return interval;
   }
 
-  const uint32_t jitter_range_ms = interval.count() * jitter_pct;
+  // divisor so the random factor below has a range of [0, floor(interval_ms * jitter_pct)]
+  const uint32_t jitter_range_ms = 1 + interval.count() * jitter_pct;
```

Comment:
Should this be rounded up? It looks like both jitter_pct and interval.count() must be greater than zero via API validations (1)(2), so probably this needs to be rounded up when converting double -> int, or min of 1, interval*pct.

Reply:
hm, yeah--i think the difference is relatively moot but using ceil here happens to make the (kinda fragile) unit test continue working, so, works for me--thanks :)